timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -23,13 +25,15 @@
 
 
 MODEL_TYPES = dict(
-    MQCNN=MQCNNModel,
-    MQRNN=MQRNNModel,
+    MQCNN=MQCNNMXNetModel,
+    MQRNN=MQRNNMXNetModel,
     SimpleFeedForward=SimpleFeedForwardModel,
+    SimpleFeedForwardMXNet=SimpleFeedForwardMXNetModel,
     DeepAR=DeepARModel,
+    DeepARMXNet=DeepARMXNetModel,
     Prophet=ProphetModel,
-    Transformer=TransformerModel,
-    TemporalFusionTransformer=TemporalFusionTransformerModel,
+    Transformer=TransformerMXNetModel,
+    TemporalFusionTransformer=TemporalFusionTransformerMXNetModel,
```

Comment:
Maybe we should add the MX suffix to all MXNet models so that users understand that enabling these requires installing MXNet? Otherwise it's not very clear right now

Reply:
This follows other conventions in Tabular (NeuralNetMXNet) and I also think it makes more sense as MXNet really qualified Model, not that it's a Transformer.